lib/yardstick/v2_client/booking_request.rb

```diff
@@ -20,6 +20,7 @@ def self.process_response(resp, extras = {})
         end
         attrs[:local_start_at] = DateTime.parse(attrs[:local_start_at]) if attrs[:local_start_at].present?
         attrs[:local_end_at] = DateTime.parse(attrs[:local_end_at]) if attrs[:local_end_at].present?
+        attrs[:earliest_date_available] = DateTime.parse(attrs[:earliest_date_available]) if attrs[:earliest_date_available].present?
```

Comment:
I think since we only care about the date this should actually be a date. It should probably be cleaned up on the Measure side to be a date...but I think it is....is it coming across with time?